pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java

```diff
@@ -179,10 +179,15 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
                     .map(Supplier::get);
 
             SimpleDateFormat dateFormat = new SimpleDateFormat("dd MMMMM yyyy");
+
+            String workspacePath = moduleVersion.getWorkspace().getPath();
+            String attributeFileRelPath = moduleVersion.getWorkspace().attributeFile().get();
             // build the attributes (default + those coming from http parameters)
             AttributesBuilder atts = AttributesBuilder.attributes()
                     // show the title on the generated html
                     .attribute("showtitle")
+                    // provide attribute file as argument to ASCIIDOCTOR for building doc.
+                    .attribute("attsFile", workspacePath + "/"+ attributeFileRelPath)
```

Comment:
You're missing a space after the "/", but I won't ask for another commit, just please keep an eye out in the future. :)

Thanks!